### PR TITLE
Remove Festivus

### LINKED ISSUE #325: Remove references to Festivus
They were put there by Atesh millennia ago, and it's time to remove them.

### PR DESCRIPTION
**Pull request by 0xazure**
Removes references to Festivus, which was disabled a while ago.

Closes #325.

### CODE REVIEW

**Comment by bvtsang** [comment]
Don't forget to bump the version.

File: Extensions/xkit_preferences.js [hunk 1]
```diff
@@ -679,60 +658,8 @@ XKit.extensions.xkit_preferences = new Object({
...
-
-				$("#xkit-festivus-toggle").html(festivus_off);
-				$("#xkit-festivus").css("display","none");
-
-				XKit.storage.set("xkit_preferences","festivus","yes");
-
-				startFestivusInterval();
-			}
-
 		});
```
Comment (bvtsang) [inline Extensions/xkit_preferences.js]: Should this line be removed as well?

Comment (0xazure) [inline Extensions/xkit_preferences.js]: Yep, looks like that's what Travis didn't like.

**Comment by 0xazure** [comment]
Bumped version and appeased Travis.  Was a little over-zealous in my deletions.

**Comment by bvtsang** [comment]
LGTM

@homu r+

**Pull request merged**